EventFilter/SiStripRawToDigi/plugins/SiStripRawToDigiUnpacker.cc

```diff
@@ -305,7 +305,13 @@ namespace sistrip {
                     << "[sistrip::RawToDigiUnpacker::" << __func__ << "]"
                     << " Invalid packet code " << buffer->packetCode(legacy_, iconn->fedCh())
                     << " for " << *ifed << " channel " << iconn->fedCh();
-                } }
+                }
+                if ( packet_code == 0 ) {
```

Comment:
Is this the only possibility for packet_code being 0?
If so, the previous LogWarning shouldn't get fired by it, i.e.. that warning should only be put after an ``else if`` statement
If not, this workaround could lead to misbehaviours in case of ``packet_code == 0`` due to some different origin

Reply:
From what I understood, packet code zero for readout mode zero suppressed is simply invalid (so there should be no other cases), which is why I would keep the warning (which is silenced unless debugging is on because it may print a message for every channel) even if we try to unpack such a channel anyway now, but that's only for backwards-compatibility with pre-2015 repacked data. It's easy to change to only issuing it `else if...`, though, if you think that's better.

Reply:
Good idea, with the last commit I added a summary of warnings (this and the other errors that were silenced in non-debug mode, even though many are quite serious) that is printed once per event, in the case of the repacked 2011 HI data that becomes
```
%MSG-w SiStripRawToDigi:  SiStripRawToDigiModule:siStripDigis  07-Jun-2018 10:30:23 CEST Run: 182124 Event: 1893722
[sistrip::RawToDigiUnpacker::createDigis] warnings:
Invalid packet code 0 for zero-suppressed data (35592)
```
Would that work? (I'm sorry, I forgot about the "once per job" when implementing this... since these mean in principle (in all except this very specific case) that some data is ignored, once per event looks like a reasonable granularity, but it's straightforward to change)